src/api/app/views/layouts/webui/webui.html.erb

```diff
@@ -68,6 +68,10 @@
 
     <%= render :partial => "layouts/webui/flash", :object => flash  %>
 
+    <% unless @hide_announcement_notification %>
+      <%= render :partial => "layouts/webui/announcement" %>
```

Comment:
Because it's a bit more than just a flash. We render a button inside, render the markdown, etc. . 
We could just have a generic notification ('There are new notifications...') here and a link to the actual ToS instead.

Reply:
> Because it's a bit more than just a flash? 

For the user? I don't think so. It's just another thing to act upon. If you don't want to intermix this then maybe something like this is a better approach:

![screenshot_2018-08-13 welcome - opensuse build service](https://user-images.githubusercontent.com/514785/44041648-9a68ce68-9f1e-11e8-8875-34cc5fcc58ee.png)

Reply:
I don't like this @hennevogel, this looks more like an exception / error message! I like how it is at the moment!

Reply:
Hmm. One is an exception and the other one is an announcement.

![screenshot from 2018-08-14 11-14-21](https://user-images.githubusercontent.com/514785/44083060-49d27fae-9fb3-11e8-876f-9246053a0b88.png)